contracts/drops/token/HolographDropERC721.sol

```diff
@@ -804,4 +794,10 @@ contract HolographDropERC721 is NonReentrant, ERC721H, IHolographDropERC721 {
       //uint256 id = chainPrepend + uint256(tokenId);
     }
   }
+
+  fallback() external override payable {
+    assembly{
+      revert(0x00, 0x00)
```

Comment:
Would it be better to revert with a more descriptive error message like `Function not supported` or `Fallback disabled` or something like that instead of an empty revert?

Reply:
Gotta save that gas lol. But ya we could add a simple `revert("Function not found")`.

Reply:
Whatever you think is best is cool with me

Reply:
Yea im kinda torn. I do like that it tells me why and i don't have to guess. Maybe it can be super short like `revert("Fn not found")`, `revert("No FN")`, or maybe `revert("¯\_(ツ)_/¯")`

Reply:
I updated it to return Function not found via the assembly block